core/service/manager_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"errors"
 	"testing"
 
+	"github.com/asaskevich/EventBus"
```

Comment:
What kind of dependency is here, I think we have own implementation of event wo external dependencies

Reply:
It doesn't seem that we have our own implementation - we're even using the interface from this external repo in `di.go`:
https://github.com/mysteriumnetwork/node/blob/master/cmd/di.go#L25

Reply:
Created an issue for this not to polute this PR with more changes (this PR is already up for 2 days!): https://github.com/mysteriumnetwork/node/issues/854